app/coffeescript/components/data/info_window.coffee

```diff
@@ -3,22 +3,42 @@
 define [
   'jquery'
   'flight/lib/component'
-  'map/components/mixins/map_utils'
+  'underscore'
 ], (
   $
   defineComponent
-  mapUtils
+  _
 ) ->
 
   infoWindowData = ->
 
     @defaultAttrs
-      pinRoute: "/map/pin/"
+      route: "/map/pin/"
+      allowed_filters: ['min_price', 'max_price']
+      refinements: {}
 
+    @formatRefinements = (results) ->
+      return "" if _.isEmpty(results.names)
+      "refinements=#{results.names.join('-')}-#{results.ids.join('+')}"
+
+    @formatFilters = (filters, names) ->
+      return "" if _.isEmpty(filters)
+      if _.isEmpty(names) then filters.join('&') else "&#{filters.join('&')}"
+
+    @queryParams = ->
+      return "" if _.isEmpty(@attr.refinements)
+      results = {names: [], ids: [], filters: []}
+      for name, obj of @attr.refinements
```

Comment:
We should use ```coffeescript for own name, obj``` here. See http://stackoverflow.com/questions/1963102/what-does-the-jslint-error-body-of-a-for-in-should-be-wrapped-in-an-if-statemen.